File: docs/components_page/components/input_group/size.jl
using DashBootstrapComponents, DashHtmlComponents

input_group = html_div([
    dbc_inputgroup([dbc_inputgrouptext("Large"), dbc_input()], size = "lg"),
    html_br(),
    dbc_inputgroup([dbc_inputgrouptext("Default"), dbc_input()]),
    html_br(),
    dbc_inputgroup([dbc_inputgrouptext("Small"), dbc_input()], size = "sm"),
]);

File: docs/components_page/components/input_group/size.R
library(dashBootstrapComponents)

input_group <- div(
  list(
    dbcInputGroup(
      list(
        dbcInputGroupText("Large"), dbcInput()
      ),
      size = "lg"
    ),
    br(),
    dbcInputGroup(
      list(
        dbcInputGroupText("Default"), dbcInput()
      )
    ),
    br(),
    dbcInputGroup(
      list(
        dbcInputGroupText("Small"), dbcInput()
      ),
      size = "sm"
    )
  )
)
